src/macaron/slsa_analyzer/checks/build_as_code_check.py

```diff
@@ -74,6 +74,20 @@ def _has_deploy_command(self, commands: list[list[str]], build_tool: BaseBuildTo
         return ""
 
     def run_check(self, ctx: AnalyzeContext, check_result: CheckResult) -> CheckResultType:
+        """Implement the check in this method.
```

Comment:
Discussion needed: Whether we should put pydoc for child class method that implements the method from the parent class. As far as I know, the pydoc can be inherited, that is why initially I didn't put the pydoc on inherited child class method (so that we are not duplicating the pydoc).

Reply:
> Discussion needed: Whether we should put pydoc for child class method that implements the method from the parent class. As far as I know, the pydoc can be inherited, that is why initially I didn't put the pydoc on inherited child class method (so that we are not duplicating the pydoc).

I think we should include the docstring in child classes for now and not silence Flake8. It helps readability of the code itself and we won't need o jump around to find the docstrings of the parent class while developing.